nengo_loihi/hardware/interface.py

```diff
@@ -690,6 +633,70 @@ def create_io_snip(self):
         self.nengo_io_snip_range = snip_range
 
 
+class HostSocket:
+    recv_retries = 10
```

Comment:
This doesn't seem to be used.

Reply:
Ah yeah, I added in the retry counting but forgot to add the condition to the while loop. Good catch!

Reply:
Done in 22bd6f67